images_config/ks-robot-slave.cfg

```diff
@@ -0,0 +1,356 @@
+sleep 5
+echo -e "\n${green}   INFO: Setup your source.list${NC}\n"
+sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" >> /etc/apt/sources.list.d/ros-latest.list'
+echo -e "\n${green}   INFO: Set up your keys${NC}\n"
+wget http://packages.ros.org/ros.key -O - | apt-key add - 
+echo -e "\n${green}   INFO: Install ROS${NC}\n"
+apt-get install python-ros-* -y -f
+apt-get install ros-indigo-rosbash -y -f
+#apt-get install ros-indigo-care-o-bot-robot -y -f # TODO include again after Debugging
+
+dpkg --configure -a
+echo "sources list:"
+cat /etc/apt/sources.list
+apt-get update
+apt-get upgrade -y
+sleep 20
+apt-get clean
+apt-get update
+aptitude install openssh-client=1:5.9p1-5ubuntu1.10
+apt-get install openssh-server -y
+sleep20
+apt-get install aptitude -y
+apt-get update
+sleep 20
+apt-get clean
+apt-get update
+apt-get upgrade -y
+apt-get -f install 
+sleep20
+
+###############################SETUP UDEV RULES############################################### 
+echo -e "\n${green}INFO: Setup udev rules${NC}\n"
+wget -O /etc/udev/rules.d/98-led.rules https://raw.githubusercontent.com/ipa320/setup_cob4/master/udev_rules/98-led.rules 
+wget -O /etc/udev/rules.d/99-gripper.rules https://raw.githubusercontent.com/ipa320/setup_cob4/master/udev_rules/99-gripper.rules 
+##############################################################################################
+
+###############################SETUP BASH ENVIROMENT##########################################
+echo -e "\n${green}INFO: Setup bash environment${NC}\n"
+sleep 5
+ROBOT=$(echo ${HOSTNAME%-*})
+echo $ROBOT
+
+if [[ "$HOSTNAME" == "$ROBOT-t"* ]]
+then
+wget -O /etc/cob.bash.bashrc https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/cob.bash.bashrc.t
+fi
+
+if [[ "$HOSTNAME" == "$ROBOT-h"* ]]
+then
+wget -O /etc/cob.bash.bashrc https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/cob.bash.bashrc.h
+fi
+
+if [[ "$HOSTNAME" == "$ROBOT-s"* ]]
+then
+wget -O /etc/cob.bash.bashrc https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/cob.bash.bashrc.s 
+fi
+
+###########
+%end
+
+
+########### SETUP LOCAL ROOT USER #################
+%post
+echo "\n${green}INFO:Setup local ROOT user${NC}\n"
+echo "Defaults rootpw" >> /etc/sudoers
+rootpw robot
+#passwd root --iscrypted $1$tEHboVqc$8dkJ2OvCfg8P3z/ru2puI0
+echo "robot-local ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+echo "robot ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+apt-get -y -q update
+sleep 20
+%end
+
+############## NFS SETUP #############
+%post --interpreter /bin/bash
+echo -e "\n${green}INFO:  Install NFS${NC}\n"
+sleep 5
+apt-get install nfs-kernel-server autofs -y -f
+HOSTNAME=$(cat /etc/hostname)
+
+SERVERNAME=$(echo ${HOSTNAME%-*}-b1)
+echo $SERVERNAME
+touch /etc/auto.direct
+
+echo "/u  -fstype=nfs4    $SERVERNAME:/" >> /etc/auto.direct
+update-rc.d autofs defaults
+service autofs restart
+modprobe nfs
+
+echo "/-  /etc/auto.direct" >> /etc/auto.master
+ping -c 1 -w 3 $SERVERNAME
```

Comment:
this line make not sense, neither the if below.

please remove this block. we do not need to check if the server is available. we just configure it, that can be done offline